sql/parse/parse.go

```diff
@@ -1397,18 +1397,19 @@ func convertInsert(ctx *sql.Context, i *sqlparser.Insert) (sql.Node, error) {
 		return nil, err
 	}
 
-	if len(i.Ignore) > 0 {
-		return nil, ErrUnsupportedSyntax.New(sqlparser.String(i))
-	}
-
 	isReplace := i.Action == sqlparser.ReplaceStr
 
 	src, err := insertRowsToNode(ctx, i.Rows)
 	if err != nil {
 		return nil, err
 	}
 
-	return plan.NewInsertInto(sql.UnresolvedDatabase(i.Table.Qualifier.String()), tableNameToUnresolvedTable(i.Table), src, isReplace, columnsToStrings(i.Columns), onDupExprs), nil
+	ignore := false
+	if strings.Contains(strings.ToLower(i.Ignore), "ignore") {
+		ignore = true
+	}
```

Comment:
This should be a `bool` in vitess rather than matching on the string. Either ignore is there or it isn't.

Change that in vitess (should be super quick), or add a TODO and get this merged in first.